src/meeting/meet/network-interception/browser/utils.ts

```
@@ -0,0 +1,115 @@
+                case 'int64':
+                    value = reader.int64()
+                    break
+                case 'varint':
+                    value = reader.uint32()
+                    break
+                case 'bytes':
+                    value = reader.bytes()
+                    break
+                case 'message':
+                    const messageDecoders = (window as any)
+                        .__networkMessageDecoders
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)